install/resources/mybb_theme.xml

```diff
@@ -4978,7 +4978,7 @@ if(use_xmlhttprequest == "1")
 </tr>]]></template>
 		<template name="member_profile_findposts" version="1807"><![CDATA[<br /><span class="smalltext">(<a href="search.php?action=finduser&amp;uid={$uid}">{$lang->find_posts}</a>)</span>]]></template>
 		<template name="member_profile_findthreads" version="1807"><![CDATA[<br /><span class="smalltext">(<a href="search.php?action=finduserthreads&amp;uid={$uid}">{$lang->find_threads}</a>)</span>]]></template>
-		<template name="member_profile_groupimage" version="120"><![CDATA[<img src="{$memperms['image']}" alt="{$usertitle}" title="{$usertitle}" /><br />]]></template>
+		<template name="member_profile_groupimage" version="1817"><![CDATA[<img src="{$memperms['image']}" alt="{$usertitle}" title="{$usertitle}" /><br />]]></template>
```

Comment:
Did you forget to make the changes to the template here or is it just an accidental version bump?